packages/devtools-reps/postcss.config.js

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const mapUrl = require("postcss-url-mapper");
+const MC_PATH = "chrome://devtools/skin/images/devtools-reps/";
+const EXPRESS_PATH = "/devtools-reps/images/";
+
+function mapUrlProduction(url, type) {
+  const newUrl = url
+    .replace("/images/open-inspector.svg", MC_PATH + "open-inspector.svg")
+    .replace("/images/jump-definition.svg", MC_PATH + "jump-definition.svg");
```

Comment:
can this be more general

Reply:
Yes, it can, but we can think about it later.
Also, with the plan to migrate all this under the debugger repo, I guess there will be changes anyway, so I'd rather not spend too much time on this at the moment, since there is a pretty small amount of images.